charts/redpanda/values.yaml

```diff
@@ -537,6 +537,10 @@ statefulset:
     fsGroup: 101
     runAsUser: 101
     fsGroupChangePolicy: OnRootMismatch
+  sideCars:
+    configWatcher:
+      enabled: true
+      resources: {}
```

Comment:
Oh! Don't we need to make sure this is in the json schema for the Redpanda controller to import?

Reply:
Thank you! that was in my todos and i completely dropped the ball.

Reply:
@joejulian the error still fails with a lower timeout unfortunately 
```
2023-05-18T13:22:53.1639354Z >>> helm install redpanda-pznmh9x5ml ct_previous_revision860339983/charts/redpanda --namespace redpanda-pznmh9x5ml --wait --values ct_previous_revision860339983/charts/redpanda/ci/12-external-cert-secrets-values.yaml --timeout 600s --set=image.tag=v22.3.17
2023-05-18T13:32:54.5034755Z Error: INSTALLATION FAILED: timed out waiting for the condition
```
on this run with the reverted timeout: https://github.com/redpanda-data/helm-charts/actions/runs/5014336900/jobs/8988471288?pr=496